src/lab10.py

```diff
@@ -138,7 +149,8 @@ def innerHTML(self, handle, s):
         self.tab.render()
 
     def cookie_string(self):
-        return self.tab.cookie_string()
+        # origin ???
```

Comment:
Do you intend to implement it before publishing this chapter?

Reply:
Yes—it's implemented in the next PR where we actually start using this.